src/Build/Resources/Strings.resx

```diff
@@ -2449,6 +2449,10 @@ Utilization:          {0} Average Utilization: {1:###.0}</value>
   <data name="SDKPathCheck_Failed" xml:space="preserve">
     <value>The directory does not exist: {0}. .NET Runtime Task Host could not be instantiated. See https://aka.ms/nettaskhost for details on how to resolve this error.</value>
   </data>
+  <data name="HostObjectFailure" xml:space="preserve">
+    <value>The task "{0}" failed to acquire its host object. {1} Contact the author of the task to report this problem.</value>
```

Comment:
Do we have the task assembly at this point? I worry that a user that gets this error won't know where the task is coming from to direct their issue, and it might help if we could point them to a dll because the dll would have some product name or information inside it.